TINY-7260: Add removal notices to plugins scheduled to be removed in 6.0 (#7040)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- The editor selection could be placed in an incorrect location when undoing or redoing changes in a document containing `contenteditable="false"` elements #TINY-7663
- Unbinding a native event handler inside the `remove` event caused an exception that blocked editor removal #TINY-7730

> ### Deprecated
> 
> - Added removal notices for the `bbcode`, `fullpage`, `legacyoutput` and `spellchecker` plugins #TINY-7260

Reviewer: ```suggestion
### Deprecated
- `bbcode`, `fullpage`, `legacyoutput` and `spellchecker` plugins have been deprecated and will be removed in the next major release #TINY-7260
```

Author: ```suggestion
### Deprecated
- The `bbcode`, `fullpage`, `legacyoutput` and `spellchecker` plugins have been deprecated and will be removed in the next major release #TINY-7260
```

- The editor selection could be placed in an incorrect location when undoing or redoing changes in a document containing `contenteditable="false"` elements #TINY-7663
- Unbinding a native event handler inside the `remove` event caused an exception that blocked editor removal #TINY-7730

### Deprecated

> - The `bbcode`, `fullpage`, `legacyoutput` and `spellchecker` plugins have been deprecated and will be removed in the next major release #TINY-7260

Reviewer: ```suggestion
- The `bbcode`, `fullpage`, `legacyoutput` and `spellchecker` plugins have been deprecated and marked for removal in the next major release #TINY-7260
```
